### PR TITLE
Fixed scrollbar darkmode

### LINKED ISSUE #465: Bug — Scroll Bars Don't Change Colors
**Describe the bug**
The scroll bars don't immediately change colors when switching between dark and light mode.

**To Reproduce**
1. Go to YACS
2. Note that the scroll bar follows the current mode's color scheme
3. Toggle to the opposite color mode
4. Scroll bar remains in the original color
5. Refresh the page
6. Scroll bar now follows the new mode 

**Expected behavior**
Scroll bar should immediately change when toggling modes

### PR DESCRIPTION
**Pull request by antmcgrann**
closes #465 

Fixes issue of scrollbar not changing colors after you switch to/from dark mode

### CODE REVIEW

File: rpi_data/modules/fetch_catalog_course_info.py [hunk 1]
```diff
@@ -273,6 +273,7 @@ def get_all_courses(self):
 
 def main():
     c = acalog_client(acalog_api_key)
+    print(acalog_api_key)
```
Comment (marchdan) [inline rpi_data/modules/fetch_catalog_course_info.py:276]: Was this print left in on purpose?

**Review by marchdan — CHANGES_REQUESTED** [review]
Using page reloads to fix this issue is bulky and does not present well on the temp deploy. You might look at how other sections of the site handle the dark mode toggle for everything without having to do a page reload.

**Comment by Viyerelu23333** [comment]
Not sure if refreshing on color change is the best way to solve it...  
And your refresh did not catch the media queries changes.

Firefox does provide a good solution to this by detecting the background of the box automatically.  Safari overwrites the UA styles, so the scrollbar is transparent and color-aware.  However, Chrome will only detect the `color-scheme` meta tag.  

Admittedly, the `color-scheme` can be `light dark` to specify both modes.  But in stylesheet, we are not using media queries to detect the dark mode, so Chrome cannot detect what mode is it currently and fallbacks to `light`.  Then the scrollbars are in light.

A possible fix to this bug is to make the `color-scheme` dynamically or change the stylesheets...

**Comment by marchdan** [comment]
@Viyerelu23333 The scroll bar wouldn't have a `fill= "currentcolor"' property, would it?

**Comment by Viyerelu23333** [comment]
> @Viyerelu23333 The scroll bar wouldn't have a `fill= "currentcolor"` property, would it?

There are CSS selectors for that.  In WebKit, use `::-webkit-scrollbar-thumb { background-color }` and `::-webkit-scrollbar-track { background-color }`.  But this breaks the UA styles.

**Comment by EthanTuttle** [comment]
Developer's fix wasn't accepted and is no longer active, closing this branch and pull request. Someone else please take over the issue (maybe @SamooMei, he's pretty good at this stuff)

**Pull request closed without merging**